.github/workflows/tests.yaml

```diff
@@ -279,7 +279,7 @@ jobs:
             - name: Run Tests
               timeout-minutes: 40
               run: |
-                  scripts/run_in_build_env.sh './scripts/tests/run_python_test.py --app out/linux-x64-all-clusters-no-ble-no-wifi-tsan-clang-test/chip-all-clusters-app --factoryreset --script-args "--log-level INFO -t 3600 --disable-test ClusterObjectTests.TestTimedRequestTimeout"'
```

Comment:
Artificially lowered the timeout to induce a failure in CI to ensure this is all behaving correctly.